### ISSUE TITLE
restrict spymasters to one per team and teams to min. 2 players

### ISSUE THREAD

**Issue by a-powelson** [kentrichards/codenames#5]
Server should prevent games from starting if 
- there is more than one spymaster per team, OR
- either team has < 2 players

**Comment by kentrichards** [comment]
yeah duet would be different; we can leave that for later though
and is it a requirement that each time has only one spymaster? if there are a lot of players it may make sense for their to be two or more collaborating

**Comment by a-powelson** [comment]
Fair point, there doesn’t necessarily need to only be one spymaster. Would we want to implement something in the future to let them collaborate then?
